src/views/Players/Players.test.js

```diff
@@ -0,0 +1,7 @@
+import { render } from '@testing-library/react';
+import Players from '../../views/Players/Players';
+
+test('renders list of Players', () => {
+  const container = render(<Players />);
```

Comment:
This test should be async with an await findyByText to properly this the component and avoid a memory leak